msg/OrientationConstraint.msg

```diff
@@ -8,10 +8,21 @@ geometry_msgs/Quaternion orientation
 # The robot link this constraint refers to
 string link_name
 
-# optional axis-angle error tolerances specified
+# Optional tolerance on the three vector components of the orientation error
 float64 absolute_x_axis_tolerance
 float64 absolute_y_axis_tolerance
 float64 absolute_z_axis_tolerance
 
+# Defines how the orientation error is calculated
+# See absolute axis tolerance below to specify the magnitude.
```

Comment:
I'm good with the content, but shouldn't this new block be moved above the `absolute_axis_tolerance` entries? This line also assumes they are below.

Reply:
Oh, good catch, I thought I changed it to above... I will fix it. But about the order:

My reasoning was that if you do not care about the parameterization you don't have to read it. I imagined the file was organized with the important (required) features at the top and the optional less common features at the bottom.

But on the other hand, it is maybe better to first mention how the tolerance values are used before allowing you to specify them.

Reply:
@felixvd I changed "below" to "above". I assume users start reading at the top of the file, and they can stop reading if they don't care about which parameterization is used. (For small tolerances, or tolerance on a single axis, this does not matter.)

Is this ok?